python/revng/daemon/graphql.py

```diff
@@ -60,6 +60,11 @@ class Produced:
     result: str
 
 
+@dataclass
+class IndexError_:
```

Comment:
Why the trailing `_`?

Reply:
`IndexError` is a builtin python exception when you do an out-of-bounds array access

Reply:
OK, I guess that calling it `CommitIndex` imprves the situation.